plugins/source/aws/resources/services/apprunner/vpc_ingress_connection.go

```diff
@@ -0,0 +1,73 @@
+// Code generated by codegen; DO NOT EDIT.
+
+package apprunner
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func VpcIngressConnection() *schema.Table {
+	return &schema.Table{
+		Name:                "aws_apprunner_vpc_ingress_connection",
+		Description:         "https://docs.aws.amazon.com/apprunner/latest/api/API_VpcIngressConnection.html",
+		Resolver:            fetchApprunnerVpcIngressConnection,
+		PreResourceResolver: getVpcIngressConnection,
+		Multiplex:           client.ServiceAccountRegionMultiplexer("apprunner"),
+		Columns: []schema.Column{
+			{
+				Name:     "region",
+				Type:     schema.TypeString,
+				Resolver: client.ResolveAWSRegion,
+			},
+			{
+				Name:     "arn",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("VpcIngressConnectionArn"),
+				CreationOptions: schema.ColumnCreationOptions{
+					PrimaryKey: true,
+				},
+			},
+			{
+				Name:     "account_id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("AccountId"),
+			},
```

Comment:
Do not merge until I verify what account this is and if it can be an outside account.